packages/transformer/src/IModelTransformer.ts

```diff
@@ -6,6 +6,7 @@
  * @module iModels
  */
 import * as path from "path";
+import { EventEmitter } from "events";
```

Comment:
my bad, but you left in the TransformerEvent stuff I experimented with when you branched from my experimental branch. Please remove all changes to this file. You can do that with:
```sh
git checkout main packages/transformer/src/IModelTransformer.ts
```